server/charges.py

```diff
@@ -48,6 +48,26 @@ def amount_to_charge(opportunity):
     return quantize(total)
 
 
+def amount_to_charge_stripe(form=None):
+    """
+    Determine the amount to charge. This depends on whether the payer agreed
+    to pay fees or not. If they did then we add that to the amount charged.
+    Stripe charges 2.2% + $0.30 on single charges and 2.2% + 0.5% + $0.30
+    on recurring charges.
+
+    https://support.stripe.com/questions/can-i-charge-my-stripe-fees-to-my-customers
+    """
+    amount = float(form["amount"])
+    if form["pay_fees_value"]:
+        if form["installment_period"] == None:
+            total = (amount + 0.30) / (1 - 0.022)
+        else:
+            total = (amount + 0.30) / (1 - 0.027)
```

Comment:
**suggestion: (blocking I think?)** I don't know that we're figuring in the `2.2% + 0.5%` on the frontend because toggling between monthly and single shows the same fees. The more I think about this the more I confuse myself, but I'd expect that fee value to be different if recurring incurs that .5% subscription fee.

Reply:
Also ran into this ☝🏼 month/year on the frontend don't change and they don't seem to match stripe values.

Reply:
Yes, you're right, I need to update the frontend to also include the new subscription fee calculation